### PR TITLE
add read faq to send tippers

### PR DESCRIPTION
**Pull request by ykris45**
i added FAQ 
#hacktobers

### CODE REVIEW

**Comment by tzarebczan** [comment]
LGTM! @filipnyquist please merge when you get a chance.

**Comment by ykris45** [comment]
@tzarebczan maybe nikoo777 as assigness sir.. thanks. just get a chance..

File: bot/modules/tipbot.js [hunk 1]
```diff
@@ -286,7 +286,7 @@ function sendLBC(bot, message, tipper, recipient, amount, privacyFlag, MultiorRo
         } else {
           let tx = txLink(txId);
           let msgtail = `
-DM me with \`${message.content.split(' ', 1)[0]}\` for command specific instructions or with \`!tips\` for all available commands`;
+DM me with \`${message.content.split(' ', 1)[0]}\` for command specific instructions or with \`!tips\` for all available commands or read our [Tipbot FAQ](https://lbry.io/faq/tipbot-discord) for a more details`;
```
Comment (nikooo777) [inline bot/modules/tipbot.js]: there is a typo. `for A more`. drop the `a` please :)

**Review by nikooo777 — CHANGES_REQUESTED** [review]
typo

**Comment by ykris45** [comment]
@nikooo777 done bro

**Review by nikooo777 — APPROVED** [review]

**Comment by filipnyquist** [comment]
LGTM!

**Pull request merged**